cli/compose/types/types.go

```diff
@@ -80,6 +80,7 @@ type ServiceConfig struct {
 	CgroupParent    string   `mapstructure:"cgroup_parent"`
 	Command         ShellCommand
 	ContainerName   string   `mapstructure:"container_name"`
+	CredentialSpec  string     `mapstructure:"credential_spec"`
```

Comment:
@friism @dnephin @diogomonica Do we want to have the raw `CredentialSpec` here or should this be embedded in `Privileges`, like in the API?